.markdownlint.json

```diff
@@ -0,0 +1 @@
+{ "default": true, "MD013": false, "MD033": false }
```

Comment:
Can you update the config file as follow??

- Rename `.markdownlint.json` into `.markdownlint-cli2.yaml` 
- Change JSON into YAML
  - as far as I touched, it seems YAML seems better than raw JSON to maintain rules
  - you need to move the configs under `config`
- Add globs/ignores

Then, we do not need set options and CIs, this will work. :) 

```bash
markdownlint-cli2 # no other options, read the default config file
```

(please update other files too)